Assets/Scripts/Tutorial-Power/TutorialGuide.cs

```diff
@@ -25,16 +25,14 @@ public class TutorialGuide : MonoBehaviour
 
     public GameEvent disableBattery;
 
-    // Disable the drain feature on the battery
-    public GameEvent disableDrain;
-
     // Enable and disable switch function
     public GameEvent enableSwitch;
     public GameEvent disableSwitch;
 
     [Header("Listening To")]
     public GameEvent switchOn;
     public GameEvent powerOn;
+    public GameEvent powerOff;
```

Comment:
This isn't set in the inspector and gives a null reference exception

Reply:
Sorry I realized I forgot to save the scene.